docker/local_postgres/0004_openledger_image_view.sql

```diff
@@ -10,7 +10,9 @@ INSERT INTO public.image_popularity_metrics (
 ) VALUES
   ('flickr', 'views', 0.85),
   ('wikimedia', 'global_usage_count', 0.85),
-  ('stocksnap', 'downloads_raw', 0.85);
+  ('stocksnap', 'downloads_raw', 0.85),
```

Comment:
We have some good links explaining the math that got pulled up recently: [1](https://github.com/cc-archive/cccatalog/issues/405#issuecomment-629233047), [2](https://github.com/cc-archive/cccatalog/pull/477). It would be a great idea to at least link to that in the code, because I at least have had to look them up several times 😅

Reply:
Yes I believe so Krystle, and I asked that same question 😅 

Great call Staci, I'll add them to the script!

Reply:
Thanks!